phalcon/mvc/micro.zep

```diff
@@ -1008,17 +1008,27 @@ class Micro extends Injectable implements \ArrayAccess
 		 */
 		if typeof returnedValue == "object" {
 			if returnedValue instanceof ResponseInterface {
-				/**
-				 * Automatically send the response
-				 */
-				 if !returnedValue->isSent() {
-				 	returnedValue->send();
-				 }
+				this->sendIfUnsent(returnedValue);
```

Comment:
You make same check twice.

```suggestion
if method_exists(response, "isSent") && !returnedValue->isSent()
```

Reply:
may need to reintegrate the method, completely overlooked that check, my bad.